docs/products/opensearch/howto/opensearch-search-and-python.rst

```diff
@@ -0,0 +1,558 @@
+
+We can check the method signature to understand the function and which parameters we'll use.  As you can see, all the parameters are optional in the ``search()`` method. Find below the method signature::
+
+  client.search: (body=None, index=None, doc_type=None, params=None, headers=None)
+
+
+To run the search queries, we'll use two of these parameters - ``index`` and ``body``:
+
+* ``index``, parameter refers to the name of the index we used to load the data. Therefore, it does not change. 
+* ``body``, parameter refers to the search query specifications. We'll modify it according to our query purpose.
+
+Lucene query and query DSL
+--------------------------
+
+OpenSearch supports the **Lucene query syntax** to perform searches by using the ``q`` parameter. The ``q`` parameter expects a string with your query specifications, for example:
+
+.. code-block:: python
+
+    client.search({
+        index: 'recipes',
+        q: 'ingredients:broccoli AND calories:(>=100 AND <200)'
+    })
+
+For users, who prefer to work with nested objects and familiar structures like JSON (equivalent to Python dictionaries), OpenSearch supports the `query domain-specific language (DSL) <https://opensearch.org/docs/latest/opensearch/query-dsl/index/>`_.
+
+For the **Query DSL**, the field ``body`` expects a dictionary object which can facilitate the construction of more complex queries depending on your use case, for example:
+
+.. code-block:: python
+
+     query_body = {
+                    "query": {
+                      "multi_match": {
+                        "query": "Garlic-Lemon",
+                        "fields": [
+                          "title",
+                          "ingredients"
+                        ]
+                      }
+                    }
+                  }
+
+In this example, we are searching for "Garlic-Lemon" across ``title`` and ``ingredients`` fields. Try out yourself using our demo::
+  
+  python search.py multi-match title ingredients Garlic-Lemon
+
+Check what comes out from this interesting combination 🧄 🍋 :
+
+.. code-block:: shell
+  
+      [
+        'Garlic-Lemon Potatoes ',
+        'Lemon Garlic Mayonnaise ',
+        'Lemon Garlic Mayonnaise ',
+        'Garlic-Lemon Croutons ',
+        'Lemon-Garlic Vinaigrette ',
+        'Lemon-Garlic Lamb Chops ',
+        'Lemon Pepper Garlic Vinaigrette ',
+        'Lemon-Garlic Baked Shrimp ',
+        'Lemon-Herb Turkey with Lemon-Garlic Gravy ',
+        'Garlic, Oregano, and Lemon Vinaigrette '
+      ]
+
+For this tutorial, we focus on the query DSL syntax to construct queries modifying the ``body`` parameter. In the method ``search()``, one of the optional fields is the ``size`` field, which is defined as the number of results returned in the search. 
+
+.. note::
+  The default value of the ``size`` field is 10.
+  
+
+We're not adjusting the ``size`` parameter in this tutorial. Therefore, we're using the default value which is 10 results per search.
```

Comment:
We said three times that we're using the default result size of ten. I think once would be fine.